src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/data-governance-optimize-ingest-guide.mdx

```diff
@@ -627,6 +627,101 @@ It's not surprising that a complex and decentralized system like Kubernetes has
 
 Depending on your observability objectives you may consider adjusting the scrape interval. The default is 15s. Now the Kubernetes cluster explorer only refreshes every 45s. If your primary use of the K8s data is to support the KCE visualizations you may consider changing your scrape interval to 20s. That change from 15s to 20s can have a substantial impact. For more details about managing this, see our [Helm integration scrape interval docs](/docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm/#scrape-interval).
 
+### Filtering Namespaces
+
+The Kubernetes Integration v3 and above allows filtering which namespaces are scraped by labelling them. By default all namespaces are scraped.
+
+We use the `namespaceSelector` in the same way Kubernetes does. In order to include only namespaces matching a label, change the `namespaceSelector` by adding the following to your `values-newrelic.yaml`, under the `newrelic-infrastructure` section:
+
+```yaml
+common:
+  config:
+    namespaceSelector:
+      matchLabels:
+        key1 : "value1"
+```
+
+In this example only namespaces with the label `newrelic.com/scrape` set to `true` will be scraped:
+
+```yaml
+global:
+  licenseKey: _YOUR_NEW_RELIC_LICENSE_KEY_
+  cluster: _K8S_CLUSTER_NAME_
+
+# ... Other settings as shown above
+
+# Configuration for newrelic-infrastructure
+newrelic-infrastructure:
+  # ... Other settings as shown above
+  common:
+    config:
+      namespaceSelector:
+        matchLabels:
+          newrelic.com/scrape: "true"
+```
+
+You can also use Kubernetes expressions to include or exclude namespaces using the following syntax:
+
+```yaml
+common:
+  config:
+    namespaceSelector:
+      matchExpressions:
+      - {key: newrelic.com/scrape, operator: NotIn, values: ["false"]}
```

Comment:
@JimHagan this line seems to have the actual example rather than the general syntax shown in the next line. How about removing this specific example line since it shows up in the next example?

Also, you could mention acceptable operators in the lead-in: "You can also use Kubernetes expressions to include or exclude namespaces. For example, you can use `In` and `NotIn` operators with this syntax:"